internal/web/templates/pages/bootstrap.html

```diff
@@ -0,0 +1,91 @@
+{{define "bootstrap-success"}}
+<div class="col-md-8 p-lg-6 mx-auto text-center">
+  <h1>It's done!</h1>
+</div>
+<div class="col-md-6 p-lg-2 mx-auto">
+  <div class="alert alert-success text-center" role="alert">
+    Congrats, Glasskube is all set up! Have fun managing packages :)
```

Comment:
I think I would rather use an emoji for this, e.g. 😊 (`&#128522;&#x1F60A;`)

Also I would add a dot after the sentence